src/components/JobCard/JobCard.tsx

```diff
@@ -1,23 +1,43 @@
+import { Link } from "react-router-dom";
 
+interface JobDetails {
+  id: number;
+  name: string;
+  category: string;
+  job_tags: string[];
+  job_description: string;
+}
+interface JobCardProps {
+  jobDetails: JobDetails;
+}
 
-const JobCard = ({ job }) => {
+const JobCard: React.FC<JobCardProps> = ({ jobDetails }) => {
   return (
```

Comment:
You can de structure out the variables early if the variables are ready to use.

`const { id, name, category, job_tags, job_description } = jobDetails`

now you don't have to repeat jobDetails.something

Reply:
Thank you for the feedback, implemented the changes